broadinstitute/pearl
# JN-437: adding UX configuration of portal configs

This makes the portal environment config page editable by superusers.  This was prompted by me accidentally copying over the participantHostname from the sandbox environment to the live environment as part of the release Friday, which took the participant UX offline.  To reset it, I had to log into the prod DB manually.  I'll be making a later PR to make it harder to accidentally overwrite settings like this one that should not be copied between environments.  But this makes those problems easier to fix in the short term, as well as making it easier to password-protect the site quickly.

![image](https://github.com/broadinstitute/juniper/assets/2800795/5e950d3c-1507-4060-9f98-20b9166dd00f)

If the user isn't a superuser, the button is disabled.  This should definitely be migrated to use Nick's new button component once that is merged

![image](https://github.com/broadinstitute/juniper/assets/2800795/e7c630e3-376d-45d6-b843-bfee5e233ce4)


TO TEST:
1. restart ApiAdminApp
2. go to https://localhost:3000/ourhealth/env/sandbox/config
3. change some settings, hit "save"
4. confirm the changes are saved, and a popup appears indicating the save was successful
